src/SDKs/Azure.Core/data-plane/Azure.Core.Tests/SocketTransport/Http.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+using Azure.Core.Buffers;
+using System;
+using System.Buffers;
+using System.Buffers.Text;
+using System.Net.Http;
+using System.Text;
+
+namespace Azure.Core.Http
+{
+    // TODO (pri 3): use real HTTP Parser (but it's in corfxlab)
```

Comment:
Why is this type here? (i.e. I forgot). I suspect it's used by the socket implementation of the transport. The socket implementation should stay in labs repo and so this type can be there.

Reply:
Filed https://github.com/Azure/azure-sdk-for-net/issues/5219 for this.